jerry-core/parser/js/js-parser-util.c

```diff
@@ -1243,6 +1243,10 @@ parser_error_to_string (parser_error_t error) /**< error code */
     {
       return "Lexical declaration cannot appear in a single-statement context";
     }
+    case PARSER_ERR_GENERATOR_IN_SINGLE_STATEMENT_POS:
+    {
+      return "Generator function is not allowed in single statement position";
```

Comment:
position -> context
(similar error message above)

Reply:
"Lexical declaration cannot appear in a single-statement context" this message is the same in v8, but the other looks like this: "Generators can only be declared at the top level or inside a block." (with a full stop at the end). Maybe there are other restrictions for generators.